INSTALL.asciidoc

```diff
@@ -25,16 +36,20 @@ sudo zypper in libtheora-devel opencv-devel fftw3-devel libsndfile-devel pkg-con
 autoreconf -f -i
 ./configure
 make
+make install
 ----------------
++
+If you do not run `make install` then you need to ensure Perl can find
+`ppmclibs/blib/arch/auto/tinycv.so` and `ppmclibs/tinycv.pm` some other way.
```

Comment:
should work out of the box if you call isotovideo where it is, at least does for me and I did not add anything special to an LD_LIBRARY_PATH or so